gtsam/hybrid/HybridNonlinearISAM.cpp

```diff
@@ -39,8 +39,8 @@ void HybridNonlinearISAM::update(const HybridNonlinearFactorGraph& newFactors,
   if (newFactors.size() > 0) {
     // Reorder and relinearize every reorderInterval updates
     if (reorderInterval_ > 0 && ++reorderCounter_ >= reorderInterval_) {
-      // TODO(Varun) Relinearization doesn't take into account pruning
-      reorder_relinearize();
+      // TODO(Varun) Re-linearization doesn't take into account pruning
+      reorderRelinearize();
```

Comment:
PS this name came from `NonlinearISAM` (the continuous only version).

Reply:
It's `NonlinearISAM::reorder_relinearize` there which is why we kept the same name.